Vagrantfile

```diff
@@ -0,0 +1,35 @@
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL     
+    sudo apt-get update 
+    # TODO: Install pyenv prerequisites  
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git   
+    # TODO: Install pyenv   
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bash_profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bash_profile
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.bash_profile
```

Comment:
It's better to update the `.profile` file instead. One already existed on the VM, whereas you have created a .bash_profile file which is now being used *instead of* the .profile, meaning you miss out on a bit of functionality that was in there. And there's a risk of adding things in the wrong place if you have both files